magma/debug.py

```diff
@@ -1,43 +1,31 @@
-import inspect
+import uinspect
 import collections
-import magma
-from magma.config import get_debug_mode
 
 
-debug_info = collections.namedtuple("debug_info", ["filename", "lineno", "module"])
+debug_info = collections.namedtuple("debug_info",
+                                    ["filename", "lineno", "module"])
```

Comment:
while we're here (`import dataclasses` above)
```suggestion
class _DebugInfo:
    filename: str
    lineno: int
    module: str
    
 
 debug_info = _DebugInfo
```